packages/shared/src/components/banners/FixedBottomBanner.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { ReactElement } from 'react';
+import CloseButton from '../CloseButton';
+import LoginButton from '../LoginButton';
+import { Button } from '../buttons/Button';
+import { tellMeWhy } from '../../lib/constants';
+
+interface FixedBottomBannerProps {
+  onDismiss(): void;
+  daysLeft: number;
+}
+
+export function FixedBottomBanner({
+  onDismiss,
+  daysLeft,
+}: FixedBottomBannerProps): ReactElement {
+  return (
+    <div className="flex fixed inset-0 z-max justify-center items-center p-6 w-full bg-gradient-to-l top-[unset] from-[#EF43FD] to-[#6451F3]">
```

Comment:
Ah no not again custom gradients 😅
This is really getting out of control

Reply:
Just gonna be a one-off thing, I promise 😂